engine/schema/src/com/cloud/upgrade/dao/Upgrade461to470.java

```diff
@@ -51,8 +53,23 @@ public boolean supportsRollingUpgrade() {
         return new File[] {new File(script)};
     }
 
+    public void alterAddColumnToCloudUsage(final Connection conn) {
+        final String alterTableSql = "ALTER TABLE `cloud_usage`.`cloud_usage` ADD COLUMN `quota_calculated` tinyint(1) DEFAULT 0 NOT NULL COMMENT 'quota calculation status'";
+        try (PreparedStatement pstmt = conn.prepareStatement(alterTableSql)) {
+            pstmt.executeUpdate();
+            s_logger.info("Altered usage table and added column quota_calculated");
+        } catch (SQLException e) {
+            if (e.getMessage().contains("quota_calculated")) {
+                s_logger.warn("Usage table already has a column called quota_calculated");
```

Comment:
The log statement to should reflect the actual name of table, cloud_usage, altered in order to avoid ambiguity.